RedisEnterprise-DNS-Migration/migration_steps.md

```diff
@@ -0,0 +1,81 @@
+# Migrate to the new Redis Enterprise Private DNS Zone
+
+This document is meant to help you migrate from an old form of the Private DNS Zone that is required to make private endpoints work with Redis Enterprise caches to a new form. Throughout this process, your infrastructure will continue to work as expected.
+
+## How to migrate to the new private DNS zone
+
+1. Identify Azure Virtual Networks (VNets) that have a private DNS zone
+    using the form:
+
+    ```{region}.privatelink.redisenterprise.cache.azure.net```
+
+2. For each of these VNets, create a new private DNS zone with the
+    name: privatelink.redisenterprise.cache.azure.net
+
+3. Establish a **Virtual network link** between your VNet and the newly
+    created zone.
+
+    - Go to the Private DNS zone page for each newly created zone and
+        click 'Virtual network links' on the
+        left.![](images/media/image1.png)
+
+    - Fill in the "Link Name" with a value of your choosing, "Virtual
+        Network" with the VNet being linked, and leave "Enable auto
+        registration" unchecked. Click **OK**.
+        ![](images/media/image2.png)
+
+4. For each private endpoint linked to that VNet, create a new A-Record
+    using the "DNS Configuration" setting on the left in the portal.
+
+    - Go to the private endpoint page for each private endpoint and click
+        "DNS Configuration" on the left.
```

Comment:
If this is is literal text on the screen, the convention is to make it Bold, not enclose in quotation marks.
```diff
-"DNS Configuration" on the left.
+ **DNS Configuration** on the left.
```